app/main.py

```diff
@@ -2,0 +1,39 @@
+from typing import List
+
+
+class Animal:
+    alive: List["Animal"] = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self._hidden: bool = False
+        Animal.alive.append(self)
+
+    @property
+    def hidden(self) -> bool:
+        return self._hidden
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, Health: {self.health}, "
+            f"Hidden: {self._hidden}}}"
+        )
```

Comment:
Code Style issue: Use interpolation instead of concatenation. It's better to use f-string formatting for building strings. It's more readable and efficient. Refer to the checklist item 'Use interpolation instead of concatenation:' for more details.